src/vs/workbench/parts/terminal/electron-browser/terminalFindWidget.ts

```diff
@@ -44,4 +47,12 @@ export class TerminalFindWidget extends SimpleFindWidget {
 	protected onFocusTrackerBlur() {
 		this._terminalService.getActiveInstance().notifyFindWidgetFocusChanged(false);
 	}
+
+	protected onFindInputFocusTrackerFocus() {
```

Comment:
@rebornix 
I really wanted to hide this further but for right now the context key has to come from above